app/routes/ui/admin_scene.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+from config import config
+from library.postgres import get_scene_from_postgres
+from models import get_session
+from routes.base import BaseUIHandler
+
+
+class UIAdminSceneHandler(BaseUIHandler):
+    """
+    Handle routes that have admin/scene/{scene_id}
+    """
+
+    async def get(self, scene_id=1):
+        try:
+            scene_id_int = int(scene_id)
+
+            self.db_session = get_session()
+            scene_dict = await get_scene_from_postgres(
+                scene_id_int,
+                self.db_session,
+                update_cache=True,
```

Comment:
Let's set update_cache=False here (the reason being, we don't touch the cache on admin routes as they may make a lot of updates in a short period of time, so no need to access or update the cache to reduce latency).